deid/dicom/pixels/clean.py

```diff
@@ -397,3 +258,163 @@ def save_dicom(self, output_folder=None, image_type="cleaned"):
             return dicom_name
         else:
             bot.warning("use detect() --> clean() before saving is possible.")
+
+
+def clean_pixel_data(
+    dicom_file: DcmOrStr,
+    results: dict,
+    fix_interpretation: bool = True,
+    pixel_data_attribute: str = "PixelData",
+) -> Optional[NDArray]:
+    """
+    Clean a dicom file.
+
+    take a dicom image and a list of pixel coordinates, and return
+    a cleaned file (if output file is specified) or simply plot
+    the cleaned result (if no file is specified)
+
+    Parameters
+    ==========
+        dicom_file: (str or FileDataset instance) Dicom file to clean
+        results: Result of the .has_burned_pixels() method
+        fix_interpretation: fix the photometric interpretation if found off
```

Comment:
Missing the pixel_data_attribute here, and the parameter should be in line with the title.